src/constants/messeges.js

```diff
@@ -0,0 +1,14 @@
+const MESSEGE = Object.freeze({
+  carNamesInput:
+    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
```

Comment:
이건 취향 차이지만, 개인적으로 줄 바꿈 기호를 더하기 연산자로 분리해서 표시하면, 줄바꿈이 포함되어 있다는 것을 조금 더 명확히 표현해줄 수 있더라구요! 이렇게 표현하는 건 어떠신가요?
```suggestion
    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)' + '\n',
```